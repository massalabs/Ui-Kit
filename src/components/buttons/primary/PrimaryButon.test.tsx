import renderer from "react-test-renderer";
import { cleanup } from "@testing-library/react";
import { PrimaryButton } from "./";

afterEach(cleanup);

describe("Primary button", () => {
  it("should render a button", () => {
    const text = `button text`;
    const onclick = jest.fn();
    const button = <PrimaryButton onCLick={onclick}>{text}</PrimaryButton>;

    // snapshot test
    const componentSnapchat = renderer.create(button);
    const tree = componentSnapchat.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
