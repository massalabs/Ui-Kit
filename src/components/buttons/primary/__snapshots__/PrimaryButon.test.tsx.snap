// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Primary button should render a button 1`] = `
<button
  className="w-64 h-48 rounded-lg text-neutral bg-primary hover:bg-tertiary"
  onClick={[MockFunction]}
  type="button"
>
  button text
  <div
    className="theme-light"
  >
    <p
      className="text-info"
    >
      information text
    </p>
  </div>
</button>
`;
